src/topology/algebra/module/multilinear.lean

```diff
@@ -217,6 +217,12 @@ lemma pi_apply {ι' : Type*} {M' : ι' → Type*} [Π i, add_comm_monoid (M' i)]
   pi f m j = f j m :=
 rfl
 
+/-- Restrict the codomain of a continuous multilinear map to a submodule. -/
+@[simps]
+def cod_restrict (f : continuous_multilinear_map R M₁ M₂) (p : submodule R M₂) (h : ∀ v, f v ∈ p) :
+  continuous_multilinear_map R M₁ p :=
+⟨f.1.cod_restrict p h, f.cont.subtype_mk _⟩
```

Comment:
If this doesn't add the lemma saying that `cod_restrict` commutes with `f.to_multilinear_map`, can you add it?

What `simps` lemma does this generate?

Reply:
I added explicit args to `simps`.